src/api/api.js

```diff
@@ -72,7 +72,7 @@ const _fetch = (
 
         _checkResponse(r, payload)
         const response = await r.json()
-        Logger.debug('Response:', response)
+        // Logger.debug('Response:', response)
```

Comment:
Yes I commented out several `Logger.debug`s because they pollute a lot the logs. I used to remove them using a stash but it adds more burden to my work in general. I'll leave them like that if you are OK with it.